packages/nextjs/src/api/index.ts

```diff
@@ -1,6 +1,65 @@
-if (!process.env.CLERK_API_KEY) {
-  throw Error('The CLERK_API_KEY environment variable must be set to use imports from @clerk/nextjs/api.');
-}
+import {
+  ClerkMiddleware,
+  ClerkMiddlewareOptions,
+  default as Clerk,
+  RequireAuthProp,
+  WithAuthProp,
+} from '@clerk/clerk-sdk-node';
+import type { NextApiRequest, NextApiResponse } from 'next';
 
-export { default } from '@clerk/clerk-sdk-node';
+export default Clerk;
 export * from '@clerk/clerk-sdk-node';
+
+// https://nextjs.org/docs/api-routes/api-middlewares#connectexpress-middleware-support
+function runMiddleware(req: NextApiRequest, res: NextApiResponse, fn: ClerkMiddleware) {
```

Comment:
🙃 
Should this go to a `utils` file since it is kind of a detail to the usage of `withAuth` etc ?

Reply:
It's a very small function specific to Next.js so I'd rather keep it here for now.

Reply:
Done.